<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图片管理器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f6f7;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="left_sidebar" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e8eef3;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>照片管理器</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_main_nav">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget {
    border: none;
    outline: 0px;
    background-color: transparent;
}
QListWidget::item {
    padding: 8px;
    border-radius: 5px;
}
QListWidget::item:selected, QListWidget::item:hover {
    background-color: #dce5ef;
}</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>全部照片</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>.</normaloff>.</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>收藏夹</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>.</normaloff>.</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>相册</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>.</normaloff>.</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_local_dir">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>本地目录</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="add_dir">
           <property name="text">
            <string>PushButton</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::ListAdd"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="tree_localdir">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_settings">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    text-align: left;
    padding: 8px;
    border: none;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #dce5ef;
}</string>
         </property>
         <property name="text">
          <string>设置</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="main_content" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="header" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_all_photos">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>全部照片</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_photo_count">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #888;</string>
            </property>
            <property name="text">
             <string>60 张照片</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_refresh">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">QScrollArea {
    border: none;
}</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>796</width>
            <height>702</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #ffffff;</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_date_1">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>2024年12月</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_month_1">
             <layout class="QGridLayout" name="gridLayout_1"/>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_date_2">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>2024年11月</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_month_2">
             <layout class="QGridLayout" name="gridLayout_2"/>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
